packages/core/src/components/popover-canvas/readme.md

```diff
@@ -12,6 +12,20 @@ use the `referenceEl` prop rather than the `selector` the referenced element can
   <h2 class="tds-headline-02 tds-u-mt0">A Popover Canvas!</h2>
 </tds-popover-canvas>
 
+```
+
+### Usage with @scania/tegel-react
+If you are using the `<TdsPopoverCanvas>` in an React environment and want to
+use the `referenceEl` prop rather than the `selector` the referenced element can't be a Tegel component. We recommend wrapping the element in a native HTMLElement and using that as the `referenceEl`. See example below:
+
+```jsx
+<div ref={myReference.current}>
```

Comment:
I wonder if this would break keyboard interaction, seeing as a plain div is not normally focusable 🤔 
Just thought of this now

Reply:
hmmm this is true, let's try it out maybe?

Reply:
Yeah seems to be working just fine

Reply:
Alright, great